### PR TITLE
fix(atomic): improved labels for rating facet values

### PR DESCRIPTION
**Pull request by btaillon**
https://coveord.atlassian.net/browse/KIT-1082

Examples:
* Rating facet
  * Inclusion filter on 5 out of 5 stars; 6112 results
  * Inclusion filter on 4 out of 5 stars; 19182 results
  * Inclusion filter on 3 out of 5 stars; 3122 results
  * Inclusion filter on 2 out of 5 stars; 619 results
  * Inclusion filter on one out of 5 stars; 572 results
* Rating range facet
  * Inclusion filter on 5 out of 5 stars; 6112 results
  * Inclusion filter on 4 to 5 stars; 24869 results
  * Inclusion filter on 3 to 5 stars; 27714 results
  * Inclusion filter on 2 to 5 stars; 28265 results
  * Inclusion filter on 1 to 5 stars; 28776 results

### CODE REVIEW

File: packages/atomic/src/locales.json [hunk 1]
```diff
@@ -1113,5 +1113,17 @@
   "query-suggestions-unavailable": {
     "en": "There are no search suggestions.",
     "fr": "Il n'y a pas de suggestions de recherche."
+  },
+  "stars": {
+    "en": "one out of {{max}} stars",
```
Comment (npushkarskii) [inline packages/atomic/src/locales.json:1118]:
```suggestion
    "en": "One out of {{max}} stars",
```

File: packages/atomic/src/locales.json [hunk 2]
```diff
@@ -1113,5 +1113,17 @@
...
+    "fr": "une étoile sur {{max}}"
```
Comment (npushkarskii) [inline packages/atomic/src/locales.json:1119]:
```suggestion
    "fr": "Une étoile sur {{max}}"
```

**Review by npushkarskii — APPROVED** [review]
LGTM, just added two minor suggestions ✌

File: packages/atomic/src/locales.json (hunk 1 above)
Comment (npushkarskii) [reply packages/atomic/src/locales.json:1118]: (pointless suggestion)

File: packages/atomic/src/locales.json (hunk 2 above)
Comment (npushkarskii) [reply packages/atomic/src/locales.json:1119]: (pointless suggestion)

**Review by npushkarskii — APPROVED** [review]

**Comment by ThibodeauJF** [comment]
It's not always stars, user can change it. Could we say something more generic, such as "rating, 5 out of 5" ?

**Review by ThibodeauJF — APPROVED** [review]

File: packages/atomic/src/locales.json [hunk 3]
```diff
@@ -1113,5 +1113,17 @@
...
+  },
+  "stars_plural": {
```
Comment (samisayegh) [inline packages/atomic/src/locales.json:1121]: This works well for English and French, but some languages have multiple plural forms (a separate word for 1 vs. 2 vs. many).

Could using [i18next's plural](https://www.i18next.com/translation-function/plurals) features be helpful in this case?


Alternatively, could placing `étoile` at the end of the phrase like in the English example remove the need for a plural key?

**Review by samisayegh — APPROVED** [review]
Not related to this PR, but there was a maintenance case recently for the JSUI on the "inclusion filter" prefix for facet values. More info available under point 3 in [this jira comment](https://coveord.atlassian.net/browse/JSUI-3315?focusedCommentId=492023).

It seems like it's preferred to simplify to just "1 out of {{max}} stars". We can wait to ask Deque about it though,

File: packages/atomic/src/locales.json (hunk 3 above)
Comment (btaillon) [reply packages/atomic/src/locales.json:1121]: In this case, having star at the beginning sounds best. It should be easy in the future when we want to add a language with multiple plural forms.

**Pull request merged**